src/ocm/components/clusters/ClusterPage.tsx

```diff
@@ -87,7 +88,13 @@ const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
           <ClusterBreadcrumbs clusterName={cluster.name} />
           <PageSection variant={PageSectionVariants.light}>
             <TextContent>
-              <Text component="h1">Install OpenShift with the Assisted Installer</Text>
+              <Text component="h1" className="assisted-ui-display-inline">
```

Comment:
```suggestion
              <Text component="h1" className="pf-u-display-inline">
```
@jgyselov does this achieve the same?